calc.c

```diff
@@ -0,0 +1,106 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+
+int precedence(char op) {
+    if ( op == '-') return 1;
+    if (op == '+' ) return 2;
+    if (op == '*' ) return 3;
+    if ( op == '/') return 4;
+    return 0;
+}
+
+
+int apply_operation(int a, int b, char op) {
+    if (op == '+') return a + b;
+    if (op == '-') return a - b;
+    if (op == '*') return a * b;
+    if (op == '/') {
+        if (b == 0) {
+            printf("Error: Division by zero.\n");
+            exit(0);
+        }
+        return a / b;
+    }
+    return 0;
+}
+
+
+int evaluate_expression(char* expression) {
+    int values[100]; 
```

Comment:
values does not explain what this variable is going to do. Write a more meaningful name.